includes/extra_datafiles/dist-site_specific_overrides.php

```diff
@@ -86,8 +86,17 @@
 
 // -----
 // Product Listing: enable sorting by column heading
-// 
+//
 // true .... Display the legacy clickable column headings to sort the product listing (in addition to the sorter dropdown)
 // false ... Do not show the column headings; this is the default.
 //
 //$show_table_header_row = false;
+
+// -----
+// Product "Additional" Images: whether to enforce "_" underscore pattern matching when not in subdirectories
+//
+// true .... Use legacy (pre v2.1.0) mode of matching "{base}{suffix}.{ext}" in main images dir but force the underscore in all subdirs
```

Comment:
So, we're going to penalize all the sites that have "followed the rules" regarding additional-image naming for the past decades when product images are present in the `/images` "root" directory?

This setting's name and default should, IMO, be reversed since those preexisting "rule-followers" are going to be wondering where the heck all their additional images disappeared to.

Reply:
Was there a rule? This bug has only just been noticed after decades!

When they notice their images have gone, it's only a one-line edit, and it flags them to sort it out.

IMHO this "inconsistency" is really a bug and so the correction should be the default.

Just put this item first on the list of the breaking changes in the update docs, maybe with a blink tag.

Reply:
> Was there a rule? This bug has only just been noticed after decades!
> 

https://github.com/zencart/documentation/blob/f680cfa9e14d3e3d151138cb8aad7b3014c7d0ae/content/user/images/image_filename_conventions.md

> When they notice their images have gone, it's only a one-line edit, and it flags them to sort it out.
> 

It's going to be a support _nightmare_.

> IMHO this "inconsistency" is really a bug and so the correction should be the default.
> 

You're entitled to your opinion.

> Just put this item first on the list of the breaking changes in the update docs, maybe with a blink tag.

Reply:
I agree with Cindy.  The default should be the current loosey-goosey, and the option should be strict.   This way a very small number of people are affected.